### PR TITLE
Changes to User Manual.

### PR DESCRIPTION
**Pull request by Manny8888**
Changes to Introduction and First Application chapters.

### CODE REVIEW

**Comment by dkochmanski** [comment]
you have mentioned on IRC, that you plan to reissue PR to take into account nyef remarks. Is it still the case? If yes, please close this PR.

**Comment by Manny8888** [comment]
I updated the files from my local repo and it looked like the pull request picked them up. If tthat's not the case, let me know. My github skills are close to nil. I might need some help.

File: Documentation/Manual/LaTeX/chap-demos-and-applications.tex [hunk 1]
```diff
@@ -20,6 +20,19 @@ \section{Running the demos}
...
+
+\begin{verbatim}
+(ql:quickload '(mcclim clim-listener))
+(clim-listener:run-listener :new-process t)
+\end{verbatim}
+
+
+Alternatively, for the more courageous, \texttt{asdf} works well starting from
+the McCLIM source code directory. 
+
```
Comment (dkochmanski) [inline Documentation/Manual/LaTeX/chap-demos-and-applications.tex]: how is this courageous? some description might be useful, like "asdf doesn't download dependencies".

File: Documentation/Manual/LaTeX/chap-demos-and-applications.tex [hunk 2]
```diff
@@ -20,6 +20,19 @@ \section{Running the demos}
...
 
+The easiest way to try this is to use the great \emph{Quicklisp}
+library manager. Assuming that this is already setup, trying out the
+demos is straightforward:
+
+\begin{verbatim}
+(ql:quickload '(mcclim clim-listener))
+(clim-listener:run-listener :new-process t)
+\end{verbatim}
+
```
Comment (dkochmanski) [inline Documentation/Manual/LaTeX/chap-demos-and-applications.tex]: I think word `great` doesn't fit in such documentation, I'd remove it.

File: Documentation/Manual/LaTeX/chap-first-application.tex [hunk 3]
```diff
@@ -15,87 +55,98 @@ \section{How CLIM applications produce output}
...
-than, say, in global variables), and to define your application-specific
-application frame in its own package.
+application frame is an instance of the class
+\class{application-frame}.  As a CLIM user, you typically define a
+class that inherits from the class \class{application-frame}, and that
+contains additional slots needed by your application.  It is
+considered good style to keep all your application-specific data in
+slots in the application frame (rather than, say, in global
+variables), and to define your application-specific application frame
+in its own package.
```
Comment (dkochmanski) [inline Documentation/Manual/LaTeX/chap-first-application.tex]: is there any change here? indentation should go into separate commit.

File: Documentation/Manual/LaTeX/chap-first-application.tex [hunk 4]
```diff
@@ -109,24 +160,27 @@ \section{A First Attempt}
...
-package as far as the user is concerned.
+contains all the symbols needed for using CLIM.  The
+\package{CLIM-LISP} package replaces the \package{COMMON-LISP} package
+for CLIM applications.  It is essentially the same as the
+\package{COMMON-LISP} package as far as the user is concerned.
 
 In our example, we export the symbol that corresponds to the main
-function to start our application, here called \texttt{APP-MAIN}.
+function to start our application, here called
+\texttt{run-my-first-app}.
```
Comment (dkochmanski) [inline Documentation/Manual/LaTeX/chap-first-application.tex]: why do we change names here?

File: Documentation/Manual/LaTeX/chap-first-application.tex (hunk 3 above)
Comment (Manny8888) [inline Documentation/Manual/LaTeX/chap-first-application.tex]: As a cop out, let me blame emacs' automatic formatting. When reviewing, formatting changed again...

File: Documentation/Manual/LaTeX/chap-first-application.tex (hunk 4 above)
Comment (Manny8888) [inline Documentation/Manual/LaTeX/chap-first-application.tex]: From personal experience when learning a new language or library, it is sometimes difficult to differentiate between what names are part of the library or something that a user should provide. Imagine something called 'display-application-main-frame'. If you know nothing about CLIM, is it part of the spec, or a user-provider name?
That's why I use that sort of very clear names to show what is what.

**Comment by Manny8888** [comment]
Aside from my comments above, I corrected what you pointed out. Changes pushed to PR.

Thanks for the review.
